streampipes-client-python/Makefile

```diff
@@ -47,7 +47,7 @@ reformat-all:
 
 .PHONY: pre-commit
 pre-commit:
-	git ls-files -- 'streampipes/**/*' | xargs poetry run pre-commit run --verbose --files
```

Comment:
We can do it this way, but since this is used as pre-commit hook I'd prefer to only scan the changed files
Scanning all files causes some unnecessary latency

Reply:
> We can do it this way, but since this is used as pre-commit hook I'd prefer to only scan the changed files Scanning all files causes some unnecessary latency

I got it. I tried replacing `git ls-files` with `git diff` in the latest commit.